CONTRIBUTING.md

```diff
@@ -1,59 +1,100 @@
+**Please read this document carefully. It will help maintainers and readers
+in solving your issue(s), evaluating your feature request, etc.**
 
-We welcome clear bug reports. If you've found a bug in Verdaccio that isn't a security risk, please file a report in our [issue tracker](https://github.com/verdaccio/verdaccio/issues). Before you file your issue, search to see if it has already been reported. If so, up-vote (using GitHub reactions) or add additional helpful details to the existing issue to show that it's affecting multiple people.
- 
-### Check if there's a simple solution in the website.
+## Development
 
-Some of the most popular topics can be found in our website(http://www.verdaccio.org/docs/en/installation.html)
+Development guides can be found on the [wiki](https://github.com/verdaccio/verdaccio/wiki):
+
+* [Building the project](https://github.com/verdaccio/verdaccio/wiki/Build-Source-Code)
+* [Running, debugging, and testing](https://github.com/verdaccio/verdaccio/wiki/Running-and-Debugging-tests)
 
-### Questions & Chat
+## Reporting Bugs
 
-We have tagged questions for easy follow up under the tag [questions](https://github.com/verdaccio/verdaccio/labels/question). Additionaly, I'd recommend to deliver questions in the new chat as **#questions/#development** channels at  [gitter](https://gitter.im/verdaccio/). 
+We welcome clear, detailed bug reports.
 
-### Using the issue tracker
+**Bugs are considered features that are not working as described in
+documentation.**
 
-The issue tracker is a channel were mostly users/developers post.
+If you've found a bug in Verdaccio **that isn't a security risk**, please file 
+a report in our [issue tracker](https://github.com/verdaccio/verdaccio/issues).
 
-#### I want to report a bug
+**NOTE: Verdaccio still does not support all npm commands. Some were not
+considered important and others have not been requested yet.**
 
-We considere a bug a feature that is not working as is described in the documentation. Before reporting a bug follow the next steps:
+### Issue Search
 
-1. Use the GitHub issue search — check if the issue has already been reported.
+Search to see if it has already been reported via
+the issue search.
 
-2. Check if the issue has been fixed — try to reproduce it using the latest master or development branch in the repository.
+Additionally, we have labelled questions for easy follow-up as [questions](https://github.com/verdaccio/verdaccio/labels/question).
 
-Verdaccio still does not support all npm commands due either in the initial design were not considered important or nobody has request it yet.
+If so, up-vote it (using GitHub reactions) or add additional helpful details to
+the existing issue to show that it's affecting multiple people.
+ 
+### Check Website For Solution
+
+Some of the most popular topics can be found in our website(http://www.verdaccio.org/docs/en/installation.html)
+
+### Chat
+
+Questions can be asked in [gitter](https://gitter.im/verdaccio/)
```

Comment:
Here we don't use **gitter** anymore, we have created a new easy link that redirect to Discord (http://chat.verdaccio.org/)